js/components/date.js

```diff
@@ -66,17 +82,24 @@ Fliplet.FormBuilder.field('date', {
 
     if (!this.value || this.autofill === 'always') {
       // HTML5 date field wants YYYY-MM-DD format
-      this.value = moment().format('YYYY-MM-DD');
+      this.value = moment().format(DATE_FORMAT);
       this.empty = false;
     }
 
+    if (this.autofill === 'empty') {
+      this.value = '';
+      this.datePicker.datepicker('setDate', '');
+
+      return;
+    }
+
     this.$emit('_input', this.name, this.value);
     $vm.$v.$reset();
   },
   watch: {
     value: function(val) {
-      if (Fliplet.Env.get('platform') === 'web') {
-        this.datePicker.datepicker('setDate', val);
+      if (!val && this.autofill !== 'empty') {
```

Comment:
Have you verified this works across all platforms? We previously had lots of issues with Upplabs making changes here.

Reply:
this has been tested on all platforms